src/functionalTest/java/uk/gov/hmcts/reform/lrdapi/client/LrdApiClient.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.hmcts.reform.lrdapi.client;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import lombok.extern.slf4j.Slf4j;
+import net.serenitybdd.rest.SerenityRest;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+
+@Slf4j
+public class LrdApiClient {
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    private static final String SERVICE_HEADER = "ServiceAuthorization";
+    private static final String AUTHORIZATION_HEADER = "Authorization";
+
+    private final String professionalApiUrl;
+    private final String s2sToken;
```

Comment:
may be make it static and call one time only though we have single api in future if we scale api's further then will useful.